client/post-editor/editor-sidebar/index.jsx

```diff
@@ -24,12 +28,22 @@ export default class EditorSidebar extends Component {
 		type: PropTypes.string,
 		toggleSidebar: PropTypes.func,
 		setPostDate: PropTypes.func,
-		isPrivate: PropTypes.bool,
+		isPostPrivate: PropTypes.bool,
 		confirmationSidebarStatus: PropTypes.string,
+		nestedSidebar: NestedSidebarPropType,
+		toggleNestedSidebar: PropTypes.func,
+	}
+
+	headerToggleSidebar = () => {
+		if ( this.props.nestedSidebar === NESTED_SIDEBAR_NONE ) {
+			this.props.toggleSidebar();
+		} else {
```

Comment:
For reference to my comment further down - see how `toggleSidebar` and `toggleNestedSidebar` act quite differently to each other when one would expect them to be the same in nature.